docs/sections/user_guide/api/index.rst

```diff
@@ -3,6 +3,7 @@ API
 
```

Comment:
Does it make sense to organize this section the same as the CLI, using headings for Drivers and Tools?

Reply:
We might not be on the same page here. I was thinking it might look nice to have something like this show up on the index page:


```
Drivers
=====
.. toctree::
chgres_cube
fv3
...

Tools
====
.. toctree::
config
file
logging
...
```

Reply:
Sorry, you're right, I was thinking of the `yaml` section.

I think maybe, if we want a hierarchy, it should first be present in the package namespaces -- e.g. `uwtools.api.config` -> `uwtools.api.tools.config` and `uwtools.api.fv3` -> `uwtools.api.drivers.fv3` -- after which we may or may not need different doc sections if items are already grouped together usefully (which I agree that at the moment they are not) and not overwhelming in number (which they are not). Maybe we should refine the `uwtools.api` namespace first? I'd be happy to do that in a quick follow-on PR.

Reply:
I think that sounds like a solid path forward. Thanks for humoring me on this one. :)

Reply:
I do see two potential downsides:
1. It will be a breaking change for API users. We probably have only one at the moment, so do you think it will be feasible to update the SRW `uwtools` `import` statements easily when it eventually updates to use a newer `uwtools` version?
2. This would also break what has been a nice 1:1 mapping between CLI modes and API modules (`uw config` -> `uwtools.api.config`, `uw fv3` -> `uwtools.api.fv3`, etc.). But it might also be that the CLI will become similarly crowded, with tools and drivers intermixed in an unhelpful way. Would we ever consider an additional CLI level, like `uw tool config ...`, `uw driver fv3 ...`, etc.? I don't love the way that looks, but looking at `uw --help` today (and thinking of all the drivers we're going to add) isn't necessarily great, either.
Can we table this for the moment and discuss it in our group session tomorrow?